src/components/Section/Section.js

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styles from './Section.scss';
+
+function getBackgroundColorClass(bgc) {
+  switch (bgc) {
+    case 'transparent':
+      return styles.isTransparent;
+    case 'white':
+      return styles.isWhite;
+    case 'grey':
+      return styles.isGrey;
+    case 'primary':
+      return styles.isPrimary;
+    case 'secondary':
+      return styles.isSecondary;
+    case 'tertiary':
+      return styles.isTertiary;
+  }
+}
```

Comment:
Maybe we can optimize this `getBackgroundColorClass` function like:

```js
function getBackgroundColorClass(bgc) {
  return bgc ? styles[`is${bgc.charAt(0).toUpperCase() + bgc.slice(1)}`] : '';
}
```

Reply:
That was the thing I was asking about generating dynamic styles! Nice! I tried a few things and none worked! Thank you so much!